src/config.js

```diff
@@ -3,4 +3,5 @@ export default {
   gaTrackingId: process.env.REACT_APP_GA_TRACKING_ID || 'UA-116475051-1',
   baseApi: process.env.REACT_APP_API_URL || 'http://localhost:3001',
   googleMaps: process.env.REACT_APP_GMAPS_URL || 'https://maps.googleapis.com/maps/api/js?key=AIzaSyCFMDqEgQ6VWWJbROzZhHu-f6sktAmTEGU&v=3.exp&libraries=geometry,drawing,places',
+  disableAuth: true
```

Comment:
What is `disableAuth` used for?

Reply:
Oops, that should not have been checked in